### PR TITLE
Added rubocop

### PR DESCRIPTION
**Pull request by ffasitella**
### Summary

- Addded rubocop
- Added .rubocop.yml to avoid two warnings

### Trello card

https://trello.com/c/DzIGt30n/8-configurar-rubocop

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Documentation:
```
Comment (marcegl91) [inline .rubocop.yml]: What does happen when you run the command that the card mentions for rubocop? ie: `bundle exec rubocop app spec -R` Could you copy here the output of that command?

File: Gemfile [hunk 2]
```diff
@@ -52,3 +52,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# Rubocop
+gem 'rubocop'
```
Comment (marcegl91) [inline Gemfile]: Please use pessimistic operator for every gem that you add to this project:

https://thoughtbot.com/blog/rubys-pessimistic-operator

File: .rubocop.yml (hunk 1 above)
Comment (ffasitella) [reply .rubocop.yml]: What does the -R param do? When I run that command, it says `invalid option: -R For usage information, use --help`. But when I run it without the -R (or just with rubocop), it works. :thinking:

Comment (marcegl91) [reply .rubocop.yml]: Exactly. The thing is the -R parameter is now deprecated, but we still need the functionality that were provided by it. The -R modifier enabled the Rails cops of Rubocop. Cops are the rules that Rubocop has for checking issues in your code:

https://github.com/rubocop-hq/rubocop/blob/master/manual/cops.md

Rails cops were cops that checked for things that were Rails-specific. So now that we don't have the -R parameter, these checks are missing, and it's up to you to configure rubocop to have again this functionality.

Comment (ffasitella) [reply .rubocop.yml]: Thank you :+1:

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+  Enabled: false
+LineLength:
+  Max: 99
+Require: rubocop-rails
```
Comment (marcegl91) [inline .rubocop.yml]: I think `require` should go all in downcase.

**Pull request merged**